scripts/skills/actives/rf_sling_smoke_bomb_skill.nut

```diff
@@ -0,0 +1,10 @@
+this.rf_sling_smoke_bomb_skill <- ::inherit("scripts/skills/actives/throw_smoke_bomb_skill", {
+	function create()
+	{
+        this.throw_smoke_bomb_skill.create();
+        this.m.Icon = "skills/rf_sling_smoke_bomb.png";
+        this.m.IconDisabled = "skills/rf_sling_smoke_bomb_sw.png";
+		this.m.Overlay = "skills/rf_sling_smoke_bomb.png";
```

Comment:
`this.m.Overlay` refers to a brush, not an image file. So this should be `rf_sling_smoke_bomb_skill`. Then we need to go in `metadata.xml` in our brush file and add an entry for this which then points to `skills/rf_sling_smoke_bomb_skill.png`

Reply:
I will just **not** overwrite the field `this.m.Overlay` an which case the icon of its parent will be used.
I think it's completely fine if those pop up icons on impact are those of the original throw-skill.

Reply:
The pop up icons are not on impact, but rather on skill use. The icon on skill use should match the icon of the skill. And it isn't a lot of work - we just need to add all the entries in metadata. The icons are already there.